tuts/start-a-private-network/v2.0.0-alpha.6/customchain.md

```diff
@@ -61,7 +61,7 @@ You should see the console outputs something as follows:
 Once your node is running, you will again notice that no blocks are being produced. At this point,
 you need to add your keys into the keystore. Remember you will need to complete these steps for each node in your network.
 
-### Option 1: Using Polkadot-JS App UI
+### Add Keys with the Polkadot-JS App UI
```

Comment:
Might flow best to remove this subheading entirely since there is no other option anymore. Totally up to you though @danforbes

Reply:
Do you think at some point there will be more than one option or do you think we will always just include this single option in this tutorial? What was the rationale for including more than one option originally?

Reply:
Not sure whether there will be another option.

I think Jimmy added the curl option originally. Maybe just for diversity? Maybe to emphasize that there isnt anything special about Apps, we're really just calling an RPC.